app/waf/test/fixtures/datasources/testrest.datasource.ts

```diff
@@ -1018,6 +1018,88 @@ export const StubResponses = {
     };
   },
 
+  bigipDOInfo200: (state: string | 'OK') => {
```

Comment:
If you want to default state to be OK, it may be: `state: string = 'OK'`

Reply:
accept it

> If you want to default state to be OK, it may be: `state: string = 'OK'`
yes. will make the change accordingly